selfdrive/boardd/panda.cc

```diff
@@ -21,9 +21,6 @@ Panda::Panda(std::string serial, uint32_t bus_offset) : bus_offset(bus_offset) {
 
   hw_type = get_hw_type();
 
-  assert((hw_type != cereal::PandaState::PandaType::WHITE_PANDA) &&
```

Comment:
Moved this check to boardd, as far as I'm aware this is not a limitation of the panda communication class.